features/support/steps_helper.rb

```diff
@@ -42,6 +42,11 @@
   all(:link, link_name).last.click
 end
 
+Then("I open the section {string}") do |section|
+  section = page.find(".govuk-details__summary-text", text: section)
```

Comment:
**v.minor nit**, do the capybara built-ins work. Just thinking that sometimes I have seen  `page.find`  fail as it may not "wait", where as built ins all have a built in "wait" timeout i believe

```
click_link('id-of-link')
click_link('Link Text')
```

Reply:
To add to this, if we really want to get serious about accessibility-first test-driven-development, we might want to look into using [this gem from the citizensadvice team](https://github.com/citizensadvice/capybara_accessible_selectors).

That would allow us to use accessible capybara selectors like [`within_disclosure`](https://github.com/citizensadvice/capybara_accessible_selectors#within_disclosurename-find_options-block) in our tests:

```ruby
within_disclosure "Disregarded benefits" do
  expect(page).to have_css("h2", text: "Other benefits")
end
```

Reply:
This gem looks great, and I think also had the added advantage of making the capybara steps more readable 💯 I would definitely be on board with adding this, using it to write new steps and maybe over time rewriting current steps that we have already